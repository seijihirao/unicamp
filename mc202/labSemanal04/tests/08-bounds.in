push a
push b
push c
push d
push e
push f
shift
shift
shift
push g
push h
push i
push j
print-first
shift
print-first
shift
print-first
shift
print-first
shift
print-first
print-last
pop
shift
print-first
print-last
shift
is-empty
exit
