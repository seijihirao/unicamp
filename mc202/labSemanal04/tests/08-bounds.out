d
e
f
g
h
j
i
i
yep
